logical/framework/backend.go

```diff
@@ -265,11 +278,21 @@ func (b *Backend) Logger() log.Logger {
 	return logformat.NewVaultLoggerWithWriter(ioutil.Discard, log.LevelOff)
 }
 
+// System returns the backend's system view.
 func (b *Backend) System() logical.SystemView {
 	return b.system
 }
 
-// This method takes in the TTL and MaxTTL values provided by the user,
+// Type returns the backend type
+func (b *Backend) Type() logical.BackendType {
```

Comment:
It'd be good to update these across the existing backends. If not in this PR, please make yourself an issue against 0.8 so it's not forgotten.